datahub-web-react/src/app/shared/ManageAccount.tsx

```diff
@@ -100,9 +91,7 @@ export const ManageAccount = ({ urn: _urn, pictureLink: _pictureLink, name }: Pr
             </MenuItem>
             <Menu.Divider />
             <MenuItem danger key="logout" tabIndex={0}>
-                <a href="/logOut" onClick={handleLogout}>
```

Comment:
So I believe that hitting this URL will invoke the backend to end the session for the IdP. I'm not sure if we can skip hitting the "/logOut" endpoint!

Reply:
@jjoyce0510 I changed the route back to `/logOut` let me know if that fixes the issue of it not hitting the backend.